### PR TITLE
Add pynwb recipe

### PR DESCRIPTION
**Pull request by dorukozturk**

### CODE REVIEW

File: recipes/pynwb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - python-dateutil
+    - six
+    - requests
+
+test:
+  imports:
+    - pynwb
+
+about:
+  home: http://github.com/{{ org }}/{{ name }}
```
Comment (synapticarbors) [inline recipes/pynwb/meta.yaml]: Just write out the url rather than using templating variables here.

File: recipes/pynwb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  license: Custom
```
Comment (synapticarbors) [inline recipes/pynwb/meta.yaml]: It looks like ` BSD-3-Clause` license. 

https://opensource.org/licenses/BSD-3-Clause

File: recipes/pynwb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+{% set branch = "dev" %}
+{% set sha256 = "e97fd13b714303ea1fc4bff1cec52a3d2eaa0ad33f45265651774af8409010fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:  https://github.com/{{ org }}/{{ name }}/archive/{{ branch }}.tar.gz
```
Comment (synapticarbors) [inline recipes/pynwb/meta.yaml]: Does `branch == dev` guarantee the correct version? It seems potentially fragile.  It seems safer to maybe use the pypi tarball:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

**Comment by dorukozturk** [comment]
@synapticarbors PTAL one more time. Thanks for the help.

**Comment by synapticarbors** [comment]
@dorukozturk -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**